apps/dashboard/src/components/workflow-editor/add-step-menu.tsx

```diff
@@ -53,7 +53,7 @@ const MenuItem = ({
       <span className="text-xs">{children}</span>
       {disabled && (
         <Badge kind="pill" variant="soft" className="ml-auto opacity-40">
-          <BadgeContent variant="neutral">soon</BadgeContent>
+          <BadgeContent variant="neutral">coming soon</BadgeContent>
```

Comment:
suggestion from: https://linear.app/novu/issue/NV-4667/[ux]-consider-replacing-the-soon-text-with-coming-soon-in-the-bubble